### DISCUSSION TITLE
getSignatureStatuses infinite loop

### DISCUSSION THREAD

**Discussion by pasztorl** [phantom/docs#305] [Help & Support] [answered]
### Summary

I'm using phantom wallet with my website. After creating some transaction, phantom stops working. If I open developer tools for the phantom wallet I see that phantom wallet trying to do thousands of getSignatureStatuses calls and receives 429 from the solana rpc.

### Example

https://4e6040.io

### Steps to Reproduce

mint substance
create artwork
hit 'inspect' at phantom wallet then check the network tab:

<img width="1628" alt="Screenshot 2024-10-22 at 20 21 33" src="https://github.com/user-attachments/assets/150782e0-bb5c-4b3b-a0b2-506467f8d848">

### Phantom Version

latest

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
This was discussed over email. It has been determined to be a known issue isolated to devnet. If anyone sees this issue on main net, please let us know.
